File: WebApp/Models/Linija.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace JGSPNSWebApp.Models
{
    public class Linija
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public string Naziv { get; set; }

        public TipLinije TipLinije { get; set; }
  
        public List<Stanica> Stanice { get; set; }

        public bool Aktivna { get; set; }



    }
}

File: AngularClientApp/src/app/Models/linija.ts
import { Stanica } from './stanica';

export class Linija 
{
    Id: number;   
    Naziv: string;
    Stanice: Stanica[];
    TipLinije: LineType;
    Aktivna: boolean;

    constructor(naziv:string, stanice:Stanica[]){
        this.Naziv = naziv;
        this.Stanice = stanice;
    }

    addLocation(stan:Stanica){
        this.Stanice.push(stan);
    }
}

export enum LineType{
    Gradski=0,
    Prigradski=1,
}